core/src/main/scala/io/finch/request/package.scala

```diff
@@ -34,6 +34,55 @@ import scala.reflect.ClassTag
 package object request {
 
   /**
+   * A reusable validation rule that can be applied to any ''RequestReader'' with a matching type.
+   */
+  class ValidationRule[T] private[request] (val description: String, predicate: T => Boolean) extends (T => Boolean) { self =>
```

Comment:
Yeah. I see. We might go with `desc` then :) I think it's pretty reasonable and short name. What do you think?